GOVERNANCE.md

```diff
@@ -43,23 +54,24 @@ depend on and trust to make decisions in the best interest of the project.
 
 Periodically, the existing maintainers curate a list of contributors that have
 shown regular activity on the project over the prior months. From this list,
-maintainer candidates are selected and proposed on the maintainers mailing list.
+maintainer candidates are selected and proposed in the maintainers forum.
 
-After a candidate has been announced on the maintainers mailing list, the
+After a candidate has been proposed in the maintainers forum, the
 existing maintainers are given five business days to discuss the candidate,
-raise objections and cast their vote. Votes may take place on the mailing list
-or via pull request comment. Candidates must be approved by at least 66% of the
-current maintainers by adding their vote on the mailing list. The reviewer role
-has the same process but only requires 33% of current maintainers. Only
-maintainers of the repository that the candidate is proposed for are allowed to
-vote.
-
-If a candidate is approved, a maintainer will contact the candidate to invite
-the candidate to open a pull request that adds the contributor to the
-MAINTAINERS file. The voting process may take place inside a pull request if a
-maintainer has already discussed the candidacy with the candidate and a
-maintainer is willing to be a sponsor by opening the pull request. The candidate
-becomes a maintainer once the pull request is merged.
+raise objections and cast their vote. Voting takes place on a pull request that
```

Comment:
s/vote/informal vote./

Reply:
s/Voting/Formal voting/

Reply:
updating `cast their vote` to `show their support`